pro_tes/middleware/task_distribution/distance.py

```diff
@@ -1,42 +1,25 @@
-"""Module for task distribution logic."""
+"""Module for distance based task distribution logic ."""
 
 from copy import deepcopy
-import random
-from urllib.parse import urlparse
-from socket import gaierror, gethostbyname
-from typing import Dict, List, Set, Tuple, Optional
 from itertools import combinations
-from ip2geotools.errors import InvalidRequestError
-from ip2geotools.databases.noncommercial import DbIpCity
-from geopy.distance import geodesic
+from socket import gaierror, gethostbyname
+from typing import Dict, List, Optional, Set, Tuple
+from urllib.parse import urlparse
 
 from flask import current_app
-import requests
-
-from pro_tes.middleware.models import AccessUriCombination, TesUriList
+from geopy.distance import geodesic
+from ip2geotools.databases.noncommercial import DbIpCity
+from ip2geotools.errors import InvalidRequestError
```

Comment:
Those should be in the second block, not the third. The third is only for imports from the `pro_tes` package (absolute or relative).

Reply:
Oh okay, probably I was thrown off by the red and green. Great then :)